executor/common.h

```diff
@@ -662,7 +662,7 @@ void execute_one(void)
 void loop(void)
 #endif
 {
-	/*SYSCALLS*/
```

Comment:
Please wither revert this file and pkg/csource/generated.go, or install a newer clang-format (6+), e.g. this one:
https://github.com/google/syzkaller/blob/master/.travis.yml#L22